akvo/rsr/models/user.py

```diff
@@ -144,9 +144,23 @@ def approved_organisations(self):
 
     def updates(self):
         """
-        return all updates created by the user
+        return all updates created by the user or by organisation users if requesting user is admin
         """
-        return ProjectUpdate.objects.filter(user=self).order_by('-created_at')
+        if self.is_superuser or self.is_admin:
+            return ProjectUpdate.objects.all().order_by('-created_at')
+        elif self.get_admin_employment_orgs():
+            owned_organisation_users = self.get_owned_org_users()
+            return ProjectUpdate.objects.filter(user__in=owned_organisation_users).order_by('-created_at')
```

Comment:
I think this could get quite costly. The ```user__in=``` part will become slow when there's > 100 users. Maybe approach this from the users' side (e.g. ```owned_organisation_users.all_updates()``` or so)?